docs/monitors/system-utilization.md

```diff
@@ -1,34 +1,44 @@
 <!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
 
-# signalfx-system-utilization
+# system-utilization
 
 
 This monitor reports utilization metrics for Windows
 Hosts.  It is used to drive the Windows Smart Agent host navigator view and dashboard
 content.
 
+Windows Performance Counters
+The underlying source for these metrics on Windows are Windows Performance Counters.
+All of the performance counters that we query in this monitor are actually Gauges
+that represent rates per second and percentages.
+This monitor reports the instantaneous values for these Windows Performance Counters.
+This means that in between a collection interval, spikes could occur on the
+Performance Counters.  The best way to mitigate this limitation is to increase
+the reporting interval on this monitor to collect more frequently.
+
 Sample YAML configuration:
 
 ```yaml
 monitors:
- - type: signalfx-system-utilization
+ - type: system-utilization
+   intervalSeconds: 2
```

Comment:
I wouldn't make this the sample because we don't have hi-res dimension handling implemented yet.